### PR TITLE
Dc/admin events

### PR DESCRIPTION
**Pull request by davemcorwin**

### CODE REVIEW

File: admin-client/src/components/Nav.svelte [hunk 1]
```diff
@@ -89,6 +89,13 @@
             </a>
           {/if}
         </li>
+        <li class="usa-nav__primary-item">
+          {#if authenticated}
```
Comment (apburnes) [inline admin-client/src/components/Nav.svelte:93]: Should we move this out to cover all of the authenticated links?

**Review by apburnes — COMMENTED** [review]
Looks good.  Threw in a question about consolidating the nav's `authenticated` if statement.

File: admin-client/src/components/Nav.svelte (hunk 1 above)
Comment (davemcorwin) [reply admin-client/src/components/Nav.svelte]: yeah, looks like they are stacked from the right so it looks ok!

Comment (davemcorwin) [reply admin-client/src/components/Nav.svelte]: done

**Review by apburnes — APPROVED** [review]
👍

**Pull request merged**